software/dittofeed.yml

```
@@ -0,0 +1,12 @@
+name: "Dittofeed"
+website_url: "https://www.dittofeed.com"
+source_code_url: "https://github.com/dittofeed/dittofeed"
+description: "Omni-channel customer engagement and messaging automation platform. Alternative to Braze, Customer.io, and Iterable."
```

We should probably follow the guidelines for alternatives we set [here](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/CONTRIBUTING.md#other-guidelines).
```suggestion
description: "Omni-channel customer engagement and messaging automation platform (alternative to Braze, Customer.io, Iterable)."
```